R/use_files.R

```diff
@@ -240,3 +241,49 @@ use_external_file <- function(
   cat_downloaded(where)
   
 }
+
+#' @export
+#' @rdname use_file
+#' @importFrom fs path_abs file_exists
+use_file <- function(
```

Comment:
Hey, 

Thanks 💪

Could you name this `use_internal_file()` please? 

Also, would you fancy making the `use_internal_css_file()`, `use_internal_js_file()` and ``use_internal_html_template()``

Reply:
Sure thing!

Looking through the external functions, I saw that you use `@importFrom fs path_abs`,  Since you also use fs::file_exists I took a chance and added that to the Roxygen skeleton (and cleaned up the `fs::` bits). If you prefer to have it reverted, let me know.

devtools::check() still results in the same two aforementioned failures.

I have only made minimal checks on the js, html, and css file handlers. I can confirm that they provide their respective messages, as well as it copies their target files to the www. js and css also successfully checks for file extensions.

Reply:
See #529. (Should I close this request then?)